modules/ossm-tutorial-bookinfo-install.adoc

```diff
@@ -69,8 +70,9 @@ You need cluster-admin rights to edit the Istio Service Mesh Member Roll.
 +
 ** Alternatively, you can run this command from the CLI to add the `bookinfo` project to the `ServiceMeshMemberRoll`. Replace `<control plane project>` with the name of your control plane project.
```

Comment:
You added underscores in the code, but not in the text ->  <control plane project>